src/common/memory.cpp

```diff
@@ -67,8 +67,7 @@ dnnl_memory::dnnl_memory(dnnl::impl::engine_t *engine,
     status_t status = engine->create_memory_storage(
             &memory_storage_ptr, flags, size, handle);
 
-    assert(status == status::success);
-    MAYBE_UNUSED(status);
+    if (status != status::success) return;
```

Comment:
Just a random place to put a comment for commit a336de93a2006586f524ab122e5e48923f14f141.

@rsdubtso is massaging the code to get it promoted.

Reply:
The patch is [promoted](9ab68b1b33319f6d996a6f156ae9c7954557a14e)